bindings/go/src/examples/perftest/perftest.go

```diff
@@ -14,10 +14,11 @@ import (
 	"sync"
 	"sync/atomic"
 	"time"
-	. "ucx"
 	"unsafe"
-	. "cuda"
 	"runtime"
+
+	. "github.com/openucx/ucx/bindings/go/src/ucx"
+	. "github.com/openucx/ucx/bindings/go/src/cuda"
```

Comment:
does it mean that when building perftest it will take ucx code from github and not local work dir?

Reply:
Yes, go import should use name matching the one in `go.mod` file, `github.com/openucx/ucx/bindings/go`.
To redirect to local work dir `go.work` and `replace` directive used.